src/main/java/mate/academy/service/UserService.java

```diff
@@ -3,18 +3,17 @@
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private static User[] users = new User[] {
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
     };
 
-    /**
-     * Find user by email. All users are stored in <code>private static final User[] users</code>
-     * @param email - the input parameter
-     * @return - user if his email is equal to passed email.
-     * Return <code>null</code> if there is no suitable user
-     */
```

Comment:
Java documentation is crucial when reading somebody's else code, so I would strongly recommend not to remove it